hasher-matcher-actioner/hmalib/common/tests/test_mappings.py

```diff
@@ -0,0 +1,62 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+import unittest
+
+from threatexchange.content_type.photo import PhotoContent
+from threatexchange.content_type.text import TextContent
+from threatexchange.content_type.video import VideoContent
+from threatexchange.signal_type.pdq import PdqSignal
+from threatexchange.signal_type.raw_text import RawTextSignal
+from threatexchange.signal_type.md5 import VideoMD5Signal
+
+from hmalib.common.config import HMAConfig
+from hmalib.common.models.tests.ddb_test_common import HMAConfigTestBase
+from hmalib.common.mappings import (
+    HMASignalTypeMapping,
+)
+
+
+class SignalTypeMappingInitTestCase(HMAConfigTestBase, unittest.TestCase):
+    def test_bootstrapped_values_work(self):
+        with self.fresh_dynamodb():
+            HMAConfig.initialize(self.__class__.table.table_name)
+            first_response = HMASignalTypeMapping.get_from_config_or_default()
+            second_response = HMASignalTypeMapping.get_from_config_or_default()
+
+            # Check that the first and second responses are the same.
+            self.assertEqual(
+                first_response.content_by_name, second_response.content_by_name
+            )
```

Comment:
Are you using pytest in HMA? I recently spent a lot of time reading through the pytest docs, and as a result understand it a lot better now. Basically unnitest.assertEqual and friends strip all the pytest helper information out of your tests, and unittest.TestCase doesn't actually get you anything that pytest can't do similarly/better (thought it may also just be functional vs object-oriented taste differences).

Reply:
I see. Time to RTFM! :)